Sources/Workspace/Workspace+Registry.swift

```diff
@@ -257,18 +257,29 @@ extension Workspace {
                     var modifiedDependencies = [TargetDescription.Dependency]()
                     for dependency in target.dependencies {
                         var modifiedDependency = dependency
-                        if case .product(let name, let packageName, let moduleAliases, let condition) = dependency,
-                           let packageName
-                        {
-                            // makes sure we use the updated package name for target based dependencies
-                            if let modifiedPackageName = targetDependencyPackageNameTransformations[packageName] {
+                        switch dependency {
+                        case let .product(name: name, package: packageName, moduleAliases: moduleAliases, condition: condition):
+                            if let packageName,
+                               // makes sure we use the updated package name for target based dependencies
+                               let modifiedPackageName = targetDependencyPackageNameTransformations[packageName] {
```

Comment:
nit: formatting seems off by one whitespace here
```suggestion
                                // makes sure we use the updated package name for target based dependencies
                                let modifiedPackageName = targetDependencyPackageNameTransformations[packageName] {
```

Reply:
This is indented with Xcode's `Editor > Structure > Re-indent`. Are you sure you'd prefer it to be re-indented as you suggested instead?

Reply:
Per [`CONTRIBUTING.md`](https://github.com/swiftlang/swift-package-manager/blob/main/CONTRIBUTING.md#creating-pull-requests) we prefer SwiftFormat instead of Xcode or swift-format in this repository. It's also very surprising that Xcode suggests 3-spaces formatting even if for a few lines. Regardless, I don't insist applying SwiftFormat here unless you can easily avoid mass reformatting the whole file to keep the diff of this PR manageable.